#! /bin/sh /usr/share/dpatch/dpatch-run
## 09_add_linux_software_map.dpatch by Daniel Kahn Gillmor <dkg@fifthhorseman.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add linux software map file

@DPATCH@

--- redir-2.2.1.orig/redir-2.2.lsm
+++ redir-2.2.1/redir-2.2.lsm
@@ -0,0 +1,11 @@
+Begin3
+Title:   redir
+Version:  2.2
+Entered-date:  15DEC1999
+Description:  TCP Port redirector (for firewalls etc).
+Keywords:  tcp port redirector bouncer proxy
+Author:   sammy@oh.verio.com
+Primary-site:  sunsite.unc.edu /pub/Linux/system/Network/daemons
+   39936 redir-2.2.tar.gz
+Copying-policy:  GPL
+End

